### DISCUSSION TITLE
how to cut down the memory used by WebView2 runtime.. it's taking 53mbs..💀

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/wry#921] [General]
how do I cut down the memory used by WebView2 runtime.. it's taking 53mbs..💀

**Comment by simonhyll** [comment]
You dont

To clarify, 53mb is barely anything, WebView2 is similar in size for your computer as running Google Chrome or Edge. You can't expect webviews to run as efficiently as they are without some space to work with
